### PR TITLE
Show errors on registration page

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@miltontony tiny review please :)

File: tuneme/templates/profiles/register.html [hunk 1]
```diff
@@ -9,17 +9,17 @@
...
-      <ul class="messages">
-          {% for message in messages %}
-          <li>{{ message }}</li>
-          {% endfor %}
-      </ul>
-    {% endif %}
+
     <form method="post" action="{% url 'molo.profiles:user_register' %}">{% csrf_token %}
         <p><strong>{% trans "Choose a username" %}</strong></p>
+        {% if form.username.errors %}
```
Comment (miltontony) [inline tuneme/templates/profiles/register.html:15]: It looks like this needs to be `{{registration_form.username}}`

**Comment by miltontony** [comment]
This will need a test

File: tuneme/templates/profiles/register.html (hunk 1 above)
Comment (smarzban) [inline tuneme/templates/profiles/register.html:15]: @miltontony I have that in line 18

**Comment by smarzban** [comment]
@miltontony It's ready for review, I've added the tests

File: tuneme/templates/profiles/register.html (hunk 1 above)
Comment (miltontony) [inline tuneme/templates/profiles/register.html:15]: Hmn.. for consistency, lets make the rest `{{form.}}``instead of using`{{registration_form}}`

**Comment by miltontony** [comment]
:+1: :cake: 
Please make sure this is added in `molo.profiles` (and Bwise/freebasics) and add an issue for `molo-gem`

**Pull request merged**